app/controllers/application_controller.rb

```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+  # before_action :current_user
+  # @current_user ||= User.find_by(id: session[:user_id])
+  def current_user
```

Comment:
- [ ] Kindly make sure there is no commented code present as it makes the code hard to read.

Reply:
Sure thing, Thank you, and consider it done